.github/workflows/promote-nightly.yml

```diff
@@ -0,0 +1,22 @@
+# Promotes a new build to the Nightly channel on a schedule.
+
+name: Promote Nightly Channel
+
+on:
+  repository_dispatch:
```

Comment:
Is this the right trigger? It is being run when you push commits to this PR

Reply:
`repository_dispatch` allows us to trigger this workflow using the GitHub API:
* https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#repository_dispatch
* https://pakstech.com/blog/github-actions-repository-dispatch/

This did make me realize that I also need to add the input params to the `workflow_dispatch`. Done 2be9e77

Reply:
You also need to prevent these runs from triggered every time you push a commit to this PR - https://github.com/ampproject/cdn-configuration/actions/workflows/promote-nightly.yml

Reply:
(that was my initial point)

Reply:
Oh, looks like I just had a syntax error there. It doesn't actually run this workflow on the PR, it just marks this as if it did because it's a PR check. Confusing UI is confusing.

Fixed in 077ab48

Reply:
Ha, looks like `workflow_dispatch` is enough! I don't know why `repository_dispatch` exists if you can trigger workflows with Octokit. Removed in eb408cd